Makefile

```diff
@@ -419,6 +419,10 @@ L3_C_UNIT_TEST_BIN  := $(BINDIR)/$(UNIT_DIR)/l3_dump.py-test
 LOC_MACRO_TEST_BIN  := $(LOC_MACRO_TEST_CPP_PROGRAM_BIN)
 SIZE_UNIT_TEST_BIN  := $(BINDIR)/$(UNIT_DIR)/size_str-test
 
+# L3-logging interfaces' performance unit-tests
+FPRINTF_PERF_UNIT_TEST_BIN  := $(BINDIR)/$(UNIT_DIR)/l3-fprintf-perf-test
+WRITE_PERF_UNIT_TEST_BIN    := $(BINDIR)/$(UNIT_DIR)/l3-write-perf-test
```

Comment:
We need two standalone API exercisers that also double-up as perf-unit tests.

The L3-logging calls are conditionally `#defined` via the `l3_log()` macro, so we need two build targets that will generate these two binaries.